apps/sap-commerce-cloud/package.json

```diff
@@ -1,66 +1,28 @@
 {
-  "name": "sap-commerce",
-  "version": "0.2.0",
-  "private": true,
-  "homepage": "./",
-  "dependencies": {
-    "@contentful/app-scripts": "^1.7.13",
-    "@contentful/app-sdk": "^4.23.0",
-    "@contentful/f36-components": "^4.68.1",
-    "@contentful/f36-icons": "^4.29.0",
-    "@contentful/f36-tokens": "^4.0.5",
-    "@contentful/field-editor-single-line": "^1.3.1",
-    "@contentful/field-editor-test-utils": "^1.4.1",
-    "@dnd-kit/core": "^6.1.0",
-    "@dnd-kit/sortable": "^8.0.0",
-    "@emotion/css": "^11.13.0",
-    "cross-env": "^7.0.3",
-    "lodash.get": "^4.4.2",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1"
-  },
+  "name": "@contentful/sap-commerce-cloud",
+  "version": "1.0.0",
   "scripts": {
-    "start": "vite",
-    "build": "vite build",
-    "test": "vitest --logHeapUsage --coverage",
-    "test:ci": "vitest",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "build": "rimraf ./build && npm run build-frontend && npm run build-app-actions",
+    "build-app-actions": "cd app-actions && npm ci && cd .. && node build-actions.js",
+    "build-frontend": "cd frontend && npm ci --include=dev && BUILD_PATH=../build npm run build",
+    "build:dev": "rimraf ./build && npm run build-frontend && npm run build-app-actions:dev",
+    "build-app-actions:dev": "cd app-actions && npm ci && cd .. && NODE_ENV=development node build-actions.js",
```

Comment:
What was the thinking here? is vite not an option for apps with app actions?

Reply:
The diff makes this look weird, but all I did here was move what had been at in the `package.json` at the root to this file: `apps/sap-commerce-cloud/frontend/package.json`, and that file now contains all of the vite stuff. Then I created a new `package.json` at the root that has scripts responsible for building the app actions and the frontend and then deploying them. Does that help explain it, or am I missing something in your question?

Reply:
no that's basically it, thanks for that :)